package-lock.json

```diff
@@ -1,6 +1,9130 @@
 {
+  "name": "GainChanger",
+  "lockfileVersion": 2,
   "requires": true,
-  "lockfileVersion": 1,
+  "packages": {
+    "": {
```

Comment:
Just curious, do you know why 9000 lines were added to this file? You didn't add any dependencies which is usually how this file gets added to

Reply:
lol I missed that. Yeah I have an idea why it happened

Reply:
I don't think that was from me, the package lock json is more than 9k lines in the main branch as well, I think I just hadn't pulled it from main yet.

Reply:
I don't think that's what it is, when I pull your branch and run npm i it removes these lines from the package-lock. Also if it were bc you didn't pull from main it wouldn't show these lines as additions in the pr since this goes off the difference from main. Try running npm i from the base directory and see if it deletes these lines